code/modules/projectiles/projectile/special.dm

```diff
@@ -119,6 +119,10 @@
 	flag = "bomb"
 	range = 3
 
+/obj/item/projectile/kinetic/upgraded
+	damage = 12

```

Comment:
like I said, any higher damage and it can two-hit people in space, which is OP. 12 ranged damage is still pretty damn good, better than a laser